hack/update-godep-licenses.sh

```diff
@@ -140,6 +140,7 @@ if ((${BASH_VERSINFO[0]}<4)); then
     echo "$ brew install md5sha1sum bash jq"
   fi
   echo
+  exit 9
```

Comment:
I might be wrong here :) why is the exit code 9 used here?

I relate 9 to SIGKILL in my mind.

Reply:
ok, I see that it's used elsewhere in the script as well. :+1: But still curious :)

Reply:
according to [1] only 1 - 2, 126 - 165, and 255  have special meanings, I pick 9 because it used in other places in the scrip  and I think is more consistent :)

[1]: http://tldp.org/LDP/abs/html/exitcodes.html

Reply:
`git blame` says @thockin added the `exit 9`, not sure why